### PR TITLE
Added vander to the torch frontend

### LINKED ISSUE #12988: torch.Tensor.median

### PR DESCRIPTION
**Pull request by minhanh29**
Closes #12988

### CODE REVIEW

**Comment by minhanh29** [comment]
close #12988

File: .gitignore [hunk 1]
```diff
@@ -27,4 +27,5 @@ with_time_logs/
 .array_api_tests_k_flag*
 internal_automation_tools/
 .vscode/*
-.idea/*
\ No newline at end of file
+.idea/*
+.DS_Store
```
Comment (hirwa-nshuti) [inline .gitignore:31]:
```suggestion
.DS_Store*
```

**Review by hirwa-nshuti — APPROVED** [review]
Hey @minhanh29 

Thanks for contributing to ivy😊
All looks good to me👍
Going to merge🙂

**Comment by minhanh29** [comment]
Thank you very much @hirwa-nshuti

**Pull request merged**